content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -69,7 +69,7 @@ Supported object types are as follows:
 
 * Pod (used for projected volume mounts, see below)
 * Secret (can be used to allow revoking a token by deleting the Secret)
-* Node (in v1.32, creating new node-bound tokens is beta, using existing node-bound tokens is GA)
+* Node (creating new node-bound tokens is GA in in v1.33+)
```

Comment:
Let's fix the “in in”; ideally we also explain how they are useful.

Something like:
```markdown
* Node (used for FIXME)
  * Creating new node-bound tokens is generally available in Kubernetes v1.33 and later.   
    For Kubernetes v1.29 to v1.32, you may need to enable a feature gate.
```

Reply:
fixed